docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -50,19 +47,93 @@ public interface FileProcessingWorkflow {
 }
 ```
 
+### `@WorkflowMethod`
+
+**@WorkflowMethod** denotes the starting point of [Workflow Execution](/docs/concepts/what-is-a-workflow-execution). Workflow Execution completes when this method completes.
```

Comment:
```suggestion
The `@WorkflowMethod` annotation identifies the method that is the starting point of the Workflow Function Execution.
The [Workflow Execution](/docs/concepts/what-is-a-workflow-execution) completes when this method completes.
```

I know we have to define Workflow Function a bit better in the concepts - but this is accurate I think.

Reply:
ok, fixed.